src/OrchardCore/OrchardCore.Modules.Abstractions/Services/IClock.cs

```diff
@@ -1,17 +1,32 @@
-﻿using System;
+using System;
 
 namespace OrchardCore.Modules
 {
-	/// <summary>
-	/// Provides the current Utc <see cref="DateTime"/>, and time related method for cache management.
-	/// This service should be used whenever the current date and time are needed, instead of <seealso cref="DateTime"/> directly.
-	/// It also makes implementations more testable, as time can be mocked.
-	/// </summary>
-	public interface IClock
+    /// <summary>
+    /// Provides the current Utc <see cref="DateTime"/>, and time related method for cache management.
+    /// This service should be used whenever the current date and time are needed, instead of <seealso cref="DateTime"/> directly.
+    /// It also makes implementations more testable, as time can be mocked.
+    /// </summary>
+    public interface IClock
```

Comment:
Should we add `DateTimeOffset LocalNow { get; }` and `GetLocalTimeZone()` and remove `ITimezoneManager`? This way we just need to import `IClock`

Reply:
Not possible, I am working on something.

Reply:
Singleton?

Reply:
Correct